src/lib/network_pool/transaction_pool.ml

```diff
@@ -308,6 +308,27 @@ struct
 
     module Batcher = Batcher.Transaction_pool
 
+    module Lru_cache = struct
```

Comment:
We deliberately deleted this, don't think we want it back! The libp2p helper already does this, so we're just hashing stuff unnecessarily